cli/rack/__init__.py

```diff
@@ -541,51 +548,46 @@ def ingest_manifest_driver(
             if not datagraphs == []:
                 clear_driver(base_url, modelgraphs, datagraphs, triple_store, triple_store_type, Graph.DATA)
 
-        if not manifest.getNodegroupsFootprint() == []:
-            delete_nodegroups_driver(manifest.getNodegroupsFootprint(), True, True, True, base_url)
-
-    # We don't override the model and datagraphs from the ingestion components unless it's for default
-    # the step itself gets to pick the target graphs
-    if default_graph:
-        targetgraph = [Url("uri://DefaultGraph")]
-    else:
-        targetgraph = None
-
-    for (step_type, step_data) in manifest.steps:
-        if StepType.DATA == step_type:
-            stepFile = base_path / step_data
-            ingest_data_driver(stepFile, base_url, targetgraph, targetgraph, triple_store, triple_store_type, False)
-        elif StepType.MODEL == step_type:
-            stepFile = base_path / step_data
-            ingest_owl_driver(stepFile, base_url, targetgraph, triple_store, triple_store_type, False)
-        elif StepType.NODEGROUPS == step_type:
-            stepFile = base_path / step_data
-            store_nodegroups_driver(stepFile, base_url)
-        elif StepType.MANIFEST == step_type:
-            stepFile = base_path / step_data
-            ingest_manifest_driver(stepFile, base_url, triple_store, triple_store_type, False, default_graph, False)
-        elif StepType.COPYGRAPH == step_type:
-            utility_copygraph_driver(base_url, triple_store, triple_store_type, step_data[0], step_data[1])
-
-    if top_level:
-        if manifest.getCopyToDefaultGraph():
-            defaultGraph = Url("uri://DefaultGraph")
-
-            if clear:
-                clear_driver(base_url, [defaultGraph], None, triple_store, triple_store_type, Graph.MODEL)
-            for graph in manifest.getModelgraphsFootprint():
-                utility_copygraph_driver(base_url, triple_store, triple_store_type, graph, defaultGraph)
-            for graph in manifest.getDatagraphsFootprint():
-                utility_copygraph_driver(base_url, triple_store, triple_store_type, graph, defaultGraph)
-
-        if manifest.getPerformEntityResolution():
-            @with_status(f'Executing entity resolution')
-            def go() -> dict:
-                return semtk3.combine_entities_in_conn(conn=sparql_connection(base_url, [defaultGraph], defaultGraph, [], triple_store, triple_store_type))
-            go()
-
-        if manifest.getPerformOptimization():
-            invoke_optimization(optimization_url)
+            if not manifest.getNodegroupsFootprint() == []:
+                delete_nodegroups_driver(manifest.getNodegroupsFootprint(), True, True, True, base_url)
+
+        for (step_type, step_data) in manifest.steps:
+            if StepType.DATA == step_type:
+                stepFile = base_path / step_data
+                ingest_data_driver(stepFile, base_url, None, None, triple_store, triple_store_type, False)
```

Comment:
`None, None` here mean that both the model graphs and the data graphs are the default graph?

Reply:
It means they aren't overridden from what's specified in their import.yaml files.

When you're using `rack data import` directly you can override those graph URLs with a CLI flag